securedrop/create-dev-data.py

```diff
@@ -66,6 +67,10 @@ def main(staging=False):
                 )
                 continue
             create_source_and_submissions(i, num_sources)
+
+        client_test = int(os.getenv('CLIENT_TEST_DATA', 0))
+        for i in range(1, client_test):
+            create_source_and_submissions(i, client_test, client_test_data=True)
```

Comment:
so you're passing `client_test` in for `source_count`? could you instead pass `num_sources` and make `CLIENT_TEST_DATA` a boolean that says whether or not special strings should be used?